nodejs-functions/src/handlers/automatic-clock-outs/clock_out.js

```diff
@@ -1,25 +1,23 @@
-const _ = require("lodash");
-const CosmosClient = require("@azure/cosmos").CosmosClient;
-const config = require("./config");
-const TimeEntry = require("./time_entry");
-const MsalClient = require("./msal_client");
-const TimeEntryDao = require("./time_entry_dao");
-const SlackClient = require("./slack_client");
-const { CLOCK_OUT_MESSAGE, CLOCK_OUT_MESSAGE_MIDNIGHT } = require("./constants");
+const _ = require('lodash');
+const CosmosClient = require('@azure/cosmos').CosmosClient;
+const config = require('./config');
+const TimeEntry = require('./time_entry');
+const MsalClient = require('./msal_client');
+const TimeEntryDao = require('./time_entry_dao');
+const SlackClient = require('./slack_client');
+const { CLOCK_OUT_MESSAGE, CLOCK_OUT_MESSAGE_MIDNIGHT } = require('./constants');
```

Comment:
Is it necesary to change between quote and single quote?

Reply:
If it's to keep a single quote standar on file it's okay also the following comments.

Reply:
Yep, it's just to keep a standard.